BottomNavigation/Nav.js

```diff
@@ -46,27 +44,30 @@ const styles = StyleSheet.create({
   },
 });
 
-const Nav = ({ title, screen, icon, badge }: Props) => (
+const Nav = ({ title, screen, icon, badge }: Props) => {
+  return (
   <NavigatonContext.Consumer>
     {
       ({ setActiveScreen, activeScreen, tintColor }) => {
-        const activeColor = activeScreen === screen ? tintColor : undefined;
+        console.log('badge', badge)
+        const activeColor = activeScreen === screen ? tintColor : 'white';
         return (
           <TouchableOpacity style={styles.container} onPress={() => setActiveScreen(screen)}>
             <Image source={icon} style={[styles.icon, { tintColor: activeColor }]} />
             <Text allowFontScaling={false} style={[styles.title, { color: activeColor }]}>
               {title}
             </Text>
-            {badge && (
+            {badge ? (
```

Comment:
&& would have been more useful as if badge didn't exist it would not have rendered, which you are trying to achieve here as well